clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -800,7 +800,7 @@ private void validateOffsetsAsync(Map<TopicPartition, SubscriptionState.FetchPos
 
             subscriptions.setNextAllowedRetry(fetchPostitions.keySet(), time.milliseconds() + requestTimeoutMs);
 
-            RequestFuture<OffsetsForLeaderEpochClient.OffsetForEpochResult> future = offsetsForLeaderEpochClient.sendAsyncRequest(node, partitionsToValidate);
+            RequestFuture<OffsetsForLeaderEpochClient.OffsetForEpochResult> future = offsetsForLeaderEpochClient.sendAsyncRequest(node, fetchPostitions);
```

Comment:
Good catch! This was missed during the initial implementation

Reply:
Thanks for the review! I’m not sure what are the next steps, but how should I get this PR merged?